### DISCUSSION TITLE
Start up time too long?

### DISCUSSION THREAD

**Discussion by Manicpressure** [prasanthrangan/hyprdots#207] [Q&A]
Specs
Intel i5-1235U
32gb RAM
nvme SSDs

It takes around 20 seconds from the time I log in, to the time waybar starts. Then another 10 seconds until firefox starts. 
With firefox and one kitty terminal open I am using 2.5gb RAM. 

I just want to gauge if this is normal? I would probably want to debloat the system if the memory usage is that high because I will need to squeeze every last drop of performance from my machine once I get this working with virtual networks. 

Update: I was able to solve the loading problem by removing xdg-desktop-portal-gnome. On journalctl -xe I could see it was consuming an inordinate amount of CPU power and there was a massive time gap between the start and end of the process. I dont know what ramifications there is to the system but I guess I may find out soon.

**Comment by prasanthrangan** [comment]
Hi, is this same as #198 ?
